doc/apiref.rst

```diff
@@ -749,11 +734,7 @@ sorting due to the internal hashtable implementation.
    Like :func:`json_object_iter_at()`, but much faster. Only works for
    values returned by :func:`json_object_iter_key()`. Using other keys
    will lead to segfaults. This function is used internally to
-   implement :func:`json_object_foreach`.
-
-   .. versionadded:: 2.3
-
-The iteration protocol can be used for example as follows::
+   implement :func:`json_object_foreach`. Example::
```

Comment:
The following code block needs to be indented with 2 more spaces, otherwise it doesn't format correctly.